src/main/java/ch/njol/skript/classes/data/SkriptClasses.java

```diff
@@ -438,7 +438,7 @@ public Timeperiod deserialize(final String s) {
 		Classes.registerClass(new ClassInfo<>(Date.class, "date")
 				.user("dates?")
 				.name("Date")
-				.description("A date is a certain point in the real world's time which can currently only be obtained with <a href='./expressions.html#ExprNow'>now</a>.",
+				.description("A date is a certain point in the real world's time which can be obtained with <a href='./expressions.html#ExprNow'>now</a>, <a href='./expressions.html#ExprUnixDate'>unix date</a> and <a href='./functions.html#date'>date()</a>.",
```

Comment:
```suggestion
				.description("A date is a certain point in the real world's time which can be obtained with <a href='./expressions.html#ExprNow'>now expression</a>, <a href='./expressions.html#ExprUnixDate'>unix date expression</a> and <a href='./functions.html#date'>date function</a>.",
```
Nite PR just nitpicking \:)